src/components/ProfileCard.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * Profile card for one veteran to be displayed in the profile gallery.
+ *
+ * @prop styles          - style override (set width)
+ * @prop veteran         - veteran object
+ * @prop currentVeteran  - currently logged in veteran obj
+ * @prop onConnect       - callback when connect pressed
+ * @prop showProfile     - callback to show profile
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  TouchableOpacity,
+} from 'react-native';
+import _ from 'underscore';
+import update from 'immutability-helper';
+
+import { margins } from '../styles/layout';
+import { colors } from '../styles/colors';
+import { fontStyles } from '../styles/fonts';
+import { APIRoutes } from '../helpers/routes/routes';
+import BaseRequester from '../helpers/requesters/BaseRequester';
+import Button from '../components/Button';
+
+
+export default class ProfileCard extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.connectWithVeteran = this.connectWithVeteran.bind(this);
+  }
+
+  connectWithVeteran(event, onSuccess, onFailure) {
```

Comment:
since we are using the same logic in the ConnectBox, we can pull this out later to a common functions file